Owl Carousel/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Owl Carousel</title>
+        <link rel="stylesheet" href="styles.css">
+        <link rel="stylesheet" href="owl.carousel.min.css">
+    </head>
+
+    <body>
+        <div class="slider">
```

Comment:
Add proper comments where ever needed. To make it easier for the user to use your template.